portfolio/src/main/webapp/index.html

```diff
@@ -16,7 +16,7 @@
   <body>
     <div class="row">
          <div id="content">
-             <img class="back-pic" src="images/eline.png" />
+             <img class="back-pic" src="images/elinefinal.png" />
```

Comment:
This is change is also in your other PR. Are both the changes in the same gt branch? If so you might get some merge errors.

Reply:
No, I created a new branch before adding the server code